src/dos/program_mount.cpp

```diff
@@ -72,6 +72,20 @@ void MOUNT::ListMounts()
 	}
 }
 
+void MOUNT::ShowUsage()
+{
+	MoreOutputStrings output(*this);
+	output.AddString(MSG_Get("PROGRAM_MOUNT_HELP_LONG"));
+#ifdef WIN32
+	output.AddString(MSG_Get("PROGRAM_MOUNT_HELP_LONG_WIN32"));
+#elif MACOSX
+	output.AddString(MSG_Get("PROGRAM_MOUNT_HELP_LONG_MACOSX"));
+#else
+	output.AddString(MSG_Get("PROGRAM_MOUNT_HELP_LONG_OTHER"));
+#endif
```

Comment:
This does the job nicely 👍🏻 Inventing a generic mechanism for only these two occurrences would be hardly warranted.